utils/utils.js

```diff
@@ -136,10 +156,17 @@ function handleError(error, analyticsInstance) {
       }
     }
     if (errorMessage && !sampleAdBlockTest) {
```

Comment:
We need to filter the script loading errors that are not triggered by the SDK here.

Reply:
What is the benefit of filtering it here not in Bugsnag?

Reply:
To avoid false-positive errors and in turn alerts.